.github/container/test-t5x.sh

```diff
@@ -14,6 +14,7 @@ usage() {
     echo "  OPTIONS                DESCRIPTION"
     echo "  -a, --additional-args  Additional gin args to pass to t5x/train.py"
     echo "  -b, --batch-size       Global batch size (REQUIRED)"
+    echo "  -c --use-gpu-configs   If provided uses contrib/gpu configs instead of top-level configs. Notably, gpu configs use adamw instead of adafactor"
```

Comment:
Perhaps call it `--use-contrib-configs` to emphasize that it is 'contribute', as GPU is always implied by our container? 😁

Reply:
Done!